requirements.txt

```diff
@@ -0,0 +1,39 @@
+app-common-python==0.2.5
+attrs==23.1.0
+boto3==1.29.3
+botocore==1.32.3
+CacheControl==0.13.1
+ccx-messaging @ git+https://github.com/RedHatInsights/insights-ccx-messaging@v3.4.2
+certifi==2023.11.17
+charset-normalizer==3.3.2
+confluent-kafka==2.3.0
+decorator==5.1.1
+defusedxml==0.7.1
+filelock==3.13.1
+fsspec==2023.10.0
+idna==3.4
+insights-core==3.2.23
+insights-core-messaging @ git+https://github.com/RedHatInsights/insights-core-messaging@1.2.2
+Jinja2==3.1.2
+jmespath==1.0.1
+jsonschema==4.20.0
+jsonschema-specifications==2023.11.1
+lockfile==0.12.2
+MarkupSafe==2.1.3
+msgpack==1.0.7
+prometheus-client==0.18.0
+py==1.11.0
+python-dateutil==2.8.2
+python-json-logger==2.0.7
+PyYAML==6.0.1
+redis==5.0.1
+referencing==0.31.0
+requests==2.31.0
+retry==0.9.2
+rpds-py==0.13.0
+s3fs==0.4.2
+s3transfer==0.7.0
+six==1.16.0
+urllib3==2.0.7
+watchtower==3.0.1
+.
```

Comment:
@epapbak where does this all deps come from?
Could it be possible to specify these deps only in the file pyproject.toml?

Reply:
We have this same structure (dependencies defined in requirements.txt and installed with pip) in all our repositories, and I think we can stick with it. If we specify the build dependencies only in the pyproject.toml, we will need a tool like poetry to build the project, and that's not a benefit IMHO

Reply:
Isn't it just a matter of moving the dependencies to the pyproject.toml file and pip would handle that?

Reply:
I moved them to the `setup.cfg` file. `pyproject.toml` is not meant for dependency handling, according to the docs  (https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/), but for holding the information related to the build system.